tracer/test/snapshots/Security.AspNetWebFormsAsmData.Classic.enableSecurity=True.__test=blocking-ips_url=_.verified.txt

```diff
@@ -0,0 +1,108 @@
+[
+  {
```

Comment:
Like the SecurityDisabled test case, I'm surprised that there are more than two distinct traceId's. Is this expected?

Reply:
See previous remark: https://github.com/DataDog/dd-trace-dotnet/pull/3610#discussion_r1081115664